litex/soc/interconnect/stream.py

```diff
@@ -465,15 +465,26 @@ def __init__(self, nbits_from, nbits_to, ratio, reverse):
 
 class _IdentityConverter(LiteXModule):
     def __init__(self, nbits_from, nbits_to, ratio, reverse):
+        # nbits_from = nbits_to, ratio = 1
         self.sink   = sink   = Endpoint([("data", nbits_from)])
         self.source = source = Endpoint([("data", nbits_to), ("valid_token_count", 1)])
         self.latency = 0
 
         # # #
 
+        if reverse:
+            # reverse order/endianness of bytes
+            if nbits_from % 8:
+                raise ValueError("When reversing ratio=1 streams, data width must be a multiple of 8 bits")
+            self.comb += source.data.eq(reverse_bytes(sink.data)),
```

Comment:
you could put this after `sink.connect(source)`
that way it overwrites the `source.data.eq(sink.data)` of `sink.connect(source)`, just like in verilog